src/collections/blog/2022-14-01-open-source-journey-gaurav-chadha/index.mdx

```diff
@@ -0,0 +1,82 @@
+<img src={Profile} className="profileImg" align="right" alt="profile-img" />
+<p>
+  Hey👋, I am Gaurav Chadha, a student from New Delhi, India with lots of dreams, ambitions, and zeal to learn and grow more each and every day.
+</p>
+<h2 id="journey">How did it all begin?</h2>
+<p>
+My journey in open source started with late-night surfing on LinkedIn.
+I saw a post about the Layer5 organization and open source which drove me to their website and{" "}<a href = "https://slack.layer5.io">Slack workspace</a>. And then, 💥 a single-click was all it took to kickstart my journey in Open Source and set in-motion a new chapter in my career development and relationships with new colleagues. 
+I would say this was my best ever decision in life till now.
+</p>
+<h2 id="firs-pr">Getting started in the community</h2>
+<p>
+Despite the warm welcome that I receive upon joining Layer5's Slack workspace, I was a bit shy to introduce myself in the{" "}<a href = "https://layer5io.slack.com/archives/C019426UBNY">#newcomers channel</a>, but then I saw so many other folks introducing themselves. My inner voice asked, "If they can do it, why can't I?", so I went ahead and introduced myself to the community. 
+The best part for newcomers in Layer5 is that not only is each individual warmly welcomed, but that
+Layer5 {" "} <a href = "https://layer5.io/community/meshmates">[MeshMates]</a> and Community Managers host a weekly meeting (see the{" "}<a href = "https://layer5.io/community/calendar">[community calendar]</a>) dedicated to the success of newcomers, helping each person find a foothold in the community. The overview provided in each weekly Newcomers meeting gives you the leg up that you need to get started with any of{" "} <a href = "https://layer5.io/projects">Layer5's projects</a> and orients you to the how the community works (the{" "} <a href = "https://layer5.io/community/handbook">community handbook</a> is an excellent resource in this regard) and who to contact in your areas of interest.
+</p>
+<p>
+  I highly recommend all newcomers to engage as much as possible in the community. You will gradually start seeing a change in yourself.
+</p>
+
+<h3 id="first-pr">Way to my First Pull Request 🏁</h3>
+<p>
+I still remember at that time I was just a beginner with Git, HTML, CSS, and JavaScript and most of the issues on the repository were going over my head. 
+So I started with a very beginner-friendly issue about changing a background of a component and who would have thought this changing background issue will eventually change my life in open source. 
+Then I made my first ever pull request in Layer5 and that got merged!
+</p>
+<p>
+Try out some good first issues here{""} <a href = "https://github.com/layer5io/layer5/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">[Layer5]</a> {""} <a href = "https://github.com/meshery/meshery/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">[Meshery]</a>💻
```

Comment:
```suggestion
Try out some good first issues here{""} <a href = "https://github.com/layer5io/layer5/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">Layer5</a> {""} ,<a href = "https://github.com/meshery/meshery/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22">Meshery</a>💻
```

Reply:
`<Link>` for intra-site hyperlinks. `<a>` for external hyperlinks